### PR TITLE
Course validation for class creation

### PR DESCRIPTION
**Pull request by ghatighorias**
When creating a new class : If selected offered_course does not have at least one student and one teacher, then class won't be created. and an error will be shown to the user

[link to notromo bug report](https://app.nostromo.io/work/course-planner/bug-class-can-be-created-for-an-offered-course-without-teacher-or-students?e21vZHVsZTp3b3JrLHByb2plY3Q6NjM2OCxjYXJkOjkxMDQwLGNhcmRfdGl0bGU6YnVnLWNsYXNzLWNhbi1iZS1jcmVhdGVkLWZvci1hbi1vZmZlcmVkLWNvdXJzZS13aXRob3V0LXRlYWNoZXItb3Itc3R1ZGVudHN9)

### CODE REVIEW

File: mix.lock [hunk 1]
```diff
@@ -1,44 +1,44 @@
-%{"bunt": {:hex, :bunt, "0.2.0", "951c6e801e8b1d2cbe58ebbd3e616a869061ddadcc4863d0a2182541acae9a38", [:mix], [], "hexpm"},
-  "canada": {:hex, :canada, "1.0.1", "da96d0ff101a0c2a6cc7b07d92b8884ff6508f058781d3679999416feacf41c5", [:mix], [], "hexpm"},
-  "canary": {:hex, :canary, "1.1.1", "4138d5e05db8497c477e4af73902eb9ae06e49dceaa13c2dd9f0b55525ded48b", [:mix], [{:canada, "~> 1.0.1", [hex: :canada, repo: "hexpm", optional: false]}, {:ecto, ">= 1.1.0", [hex: :ecto, repo: "hexpm", optional: false]}, {:plug, "~> 1.0", [hex: :plug, repo: "hexpm", optional: false]}], "hexpm"},
-  "certifi": {:hex, :certifi, "1.0.0", "1c787a85b1855ba354f0b8920392c19aa1d06b0ee1362f9141279620a5be2039", [:rebar3], [], "hexpm"},
+%{"bunt": {:hex, :bunt, "0.2.0", "951c6e801e8b1d2cbe58ebbd3e616a869061ddadcc4863d0a2182541acae9a38", [:mix], []},
```
Comment (ggpasqualino) [inline mix.lock]: could you please run `mix local.hex --force`, then `mix deps.get` and check this file again?

Comment (ghatighorias) [inline mix.lock]: what does it do ?

Comment (ggpasqualino) [inline mix.lock]: it updates your local hex version

Comment (ghatighorias) [inline mix.lock]: it did the job

**Pull request merged**